### ISSUE TITLE
Use of undeclared Var breakout.game/block-attrs

### ISSUE THREAD

**Issue by bzg** [lambdaisland/breakout#1]
In the `re-frame-start` branch, when I first try to run the application, I get this warning:

```
30       [color-swatch idx])
31     [selection-marker 0]])
32  
33  (defn block [[x y color]]
34    [:rect (g/block-attrs x y color)])
              ^--- Use of undeclared Var breakout.game/block-attrs
35  
36  (defn blocks []
37    (let [blocks [[3 5 0] [2 8 1] [1 1 4]]]
38      [:g (map block blocks)]))
```

... and the page stays blank. `block-attrs` isn't defined anywhere AFAICT.

**Comment by lambdaisland** [comment]
My apologies, it seems this start commit was not in a good state. I've updated both the `re-frame-start` and `re-frame-episode-1` branches to more closely follow the episode. I've also updated the README to provide some more info, and added a note to the show notes, since [it is no longer necessary to use form-2 components with re-frame](https://lambdaisland.com/blog/11-02-2017-re-frame-form-1-subscriptions)

What happened was I went through a few iterations of this code to figure out a structure that would be easy and clear to explain. Somehow the views in that start branch were those from a different version of the code. The version you see in the episode does not use `g/block-attrs`.

Thanks for reporting this!
